scripts/Synapse/convertParquetMPI.ipynb

```diff
@@ -152,46 +207,59 @@
         "        pre_write_data = {\n",
         "            'blob': data[\"bundle\"],\n",
         "            'cloud_provider': 'azure',\n",
         "            'bucket_name': snapshot_bucket_name,\n",
-        "            'file_name': f\"/pre/lac_extract_{str(curr_date)}_{str(file_idx)}.json\",\n",
+        "            'file_name': pre_filename,\n",
         "            'storage_account_url':storage_account_url\n",
         "\n",
         "        }\n",
-        "\n",
-        "        resp = request_with_token_refresh(url=write_to_storage_url,headers=ingestion_headers,data=pre_write_data,endpoint_type=\"ingestion\",endpoint_login_url=endpoint_login_url)\n",
+        "        mssparkutils.fs.put(pre_filename, json.dumps(data['bundle']), True)\n",
+        "        # azure_connection_manager.upload_object(message=data[\"bundle\"],container_name=snapshot_bucket_name,filename=pre_filename)\n",
+        "        # print(file_idx, \"pre_write_to_storage\")\n",
+        "        \n",
         "\n",
         "        # Perform pipeline standardization on extracted data to mirror how non-extracted data gets handled by the pipeline\n",
-        "        resp = request_with_token_refresh(url=std_names_url,headers=ingestion_headers,data={\"data\":data[\"bundle\"]},endpoint_type=\"ingestion\",endpoint_login_url=endpoint_login_url)\n",
-        "        data['bundle'] = resp.json().get(\"bundle\")\n",
+        "        data['bundle'] = standardize_names(data = data[\"bundle\"])\n",
+        "        print(file_idx, \"std_names\")\n",
         "\n",
-        "        resp = request_with_token_refresh(url=std_phones_url,headers=ingestion_headers,data={\"data\":data[\"bundle\"]},endpoint_type=\"ingestion\",endpoint_login_url=endpoint_login_url)\n",
-        "        data['bundle'] = resp.json().get(\"bundle\")\n",
+        "        data['bundle'] = standardize_phones(data = data[\"bundle\"])\n",
+        "        print(file_idx, \"std_phones\")\n",
         "\n",
-        "        resp = request_with_token_refresh(url=std_dob_url,headers=ingestion_headers,data={\"data\":data[\"bundle\"]},endpoint_type=\"ingestion\",endpoint_login_url=endpoint_login_url)\n",
-        "        data['bundle'] = resp.json().get(\"bundle\")\n",
+        "        try:\n",
+        "            data['bundle'] = standardize_dob(data = data['bundle'])\n",
+        "            print(file_idx, \"std_dob\")\n",
+        "        except Exception as e:\n",
+        "            continue\n",
         "\n",
-        "        resp = requests.post(std_address_url, headers=ingestion_headers, json={\"bundle\":data[\"bundle\"],\"geocode_method\":\"smarty\"})\n",
-        "        data['bundle'] = resp.json().get(\"bundle\")\n",
+        "        data['bundle'] = geocode_client.geocode_bundle(bundle = data['bundle'])\n",
+        "        print(file_idx, \"std_address\")\n",
```

Comment:
The data is going to run through the entire pipeline so we don't need to do these standardizations here. We can just let the pipeline take care of things.